R/utility.R

```diff
@@ -243,3 +243,15 @@ readMetadataBlob <- function(jobId) {
 areShallowEqual <- function(a, b) {
   !is.null(a) && !is.null(b) && a == b
 }
+
+hasDataSet <- function(list) {
+  if (length(list) > 0) {
+    for (arg in list[[1]]) {
+      if (typeof(arg) == "list") {
```

Comment:
Is 'list' the only type that would be a data set? Could it be something else like a data frame or something?

Reply:
Using the iterator as an argument, data sets will be a list